public.mk

```diff
@@ -0,0 +1,19 @@
+GORELEASER_VERSION  ?= v0.138.0
+GORELEASER_OPTS     ?= --release-notes <(echo "See the [CHANGELOG](https://$(PACKAGE)/blob/v$(VERSION)/CHANGELOG.md) for details.")
+
+.PHONY: installgoreleaser
+.ONESHELL:
+installgoreleaser: ## Installs GoReleaser (https://goreleaser.com/)
+	if [ ! -f $(shell go env GOPATH)/bin/goreleaser ]; then
+		curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | \
+			sh -s -- -b $(shell go env GOPATH)/bin $(GORELEASER_VERSION)
+	fi
+
+.PHONY: release
```

Comment:
You most likely miss a `.ONESHELL:` here

Reply:
Added.